packages/devtools/web/index.html

```diff
@@ -36,13 +36,51 @@
                 );
             }
         })();
+
+        // Plotly wrappers.
+        function hookupPlotlyClick(domName, jsFunction) {
```

Comment:
Writing binding code in JS is a good idea for getting started using a js library. 
Here are some ways you could eliminate these JS shims:

you could make hookupPlotlyClick  work in pure dart code as follows:
e.g.
```dart
import 'package:js/js_util.dart' as js_util;

Element element = document.querySelector('#$domName);
js_util.callMethod(element, 'on', ['plotly_click', allowInterop(dartFn)]);
```